packages/amplify_core/lib/src/types/query/query_field_operators.dart

```diff
@@ -156,8 +181,7 @@ class LessOrEqualQueryOperator<T extends Comparable>
     if (other == null) {
       return false;
     }
-    dynamic serializedValue = serializeDynamicValue(value);
```

Comment:
Note: The addition of `toMap()` means that `other` will no longer be serialized, so `value` no longer needs to be serialized.